src/MUP_1/Kugel.java

```diff
@@ -1,10 +1,26 @@
 package MUP_1;
 
+/**
+ * NUR INFORMATIV
+ *
+ * Mit Java 16 + 17 kommen gibt es Datenklassen namens record. Diese können z.B. so aussehen:
+ * public record Kugel(double radius)
+ * der record erzeugt automatisch den Standardkonstruktor und die getter und setter für radius
+ *
+ * Normalerweiße würde man nur das Kugel Object erzeugen, ohne das diese Logik enthält.
+ * dadurch würde z.b. der record alleine reichen
+ * danach wird noch eine KugelService.class erstellt oder ähnlicher name. die den Logikpart übernimmt (also die berechnungen)
+ * die getter und setter sind aber weiterhin teil des Kugelmodels
+ *
+ * Um Projekte schnell zu initialisieren und zu bauen und testen ,..... gibt es project management tools, wie z.b. gralde und maven
+ * Dadurch muss man die ganze struktur nicht selber aufbauen, sondern wird für die generiert. Maven ist hierbei ein wenig einsteigerfreundlicher
```

Comment:
maven:
https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html

maven erlaubt auch das schnelle hinzufügen von anderen Projekten. 
Hier im Maven Repo kann man so sehen was es alles gibt:
https://mvnrepository.com/

z.B. wenn es mal um testing geht, kann man einfach nach testingframworks suchen:
z.B. https://mvnrepository.com/artifact/junit/junit

Wenn man eine Version dann auswählt wird dir der Code für mvn bereitgestellt:
<dependency>
    <groupId>junit</groupId>
    <artifactId>junit</artifactId>
    <version>4.13.2</version>
    <scope>test</scope>
</dependency>
Und schon kann man junit fürs testen benutzen (keine Ahnung wie gut man das jetzt verstehen kann, ist aber eh nur als info)